projects/Michigan_2023_Riles.yaml

```diff
@@ -0,0 +1,7 @@
+Department: Physics
+Description: 
+FieldOfScience: Astrophysics
+FieldOfScienceID: '40.0202'
+InstitutionID: https://osg-htc.org/iid/4ocf9kvq30fn
+Organization: University of Michigan?nn Arbor 
```

Comment:
@mwestphall , can you take a look at this? Looks like some unicode got added to the DB for this one.

Reply:
The DB should support unicode (the UW Madison entry should technically use an en-dash). The question is do we care? If the GRACC is using the `InstitutionID` to look up the organization name, I don't think there are any other consumers of `Organization`.

Reply:
We'll need to fix either the database entry or the YAML generation since it looks like the encoding of the dash got screwed up _somewhere_.

Reply:
It looks ok in the institution DB UI so I think that means that this is an issue with the YAML generation. Does that sound right to you @CannonLock @mwestphall ?